Packs/CrowdStrikeFalcon/Playbooks/playbook-CrowdStrike_Falcon_-_False_Positive_Incident_Handling.yml

```diff
@@ -827,4 +853,4 @@ inputs:
   playbookInputQuery:
 outputs: []
 tests:
- -  No tests
+- Test - CrowdStrike Falcon
```

Comment:
should this test be here ?

Reply:
Ill remove it since Im building a test playbook for this playbook anyway...